js/stringProcessing/checkNofollow.js

```diff
@@ -0,0 +1,17 @@
+/** @module stringProcessing/checkNofollow */
+
+/**
+ * Checks if a links has a nofollow attribute. If it has, returns Nofollow, otherwise Dofollow.
+ *
+ * @param {string} text
+ * @returns {string} Returns Dofollow or Nofollow.
```

Comment:
We should consider if this is an API we like. There are a few issues with this:
1. `rel=` can contain much more than only `nofollow`
2. Do we really want to return a string? If we call this function `hasLinkNoFollow` and return a boolean we have a much clearer API.

Let's discuss this.